dspace-xmlui/src/main/webapp/i18n/messages.xml

```diff
@@ -1024,6 +1029,7 @@
         <message key="xmlui.administrative.Navigation.context_export_collection">Export Collection</message>
         <message key="xmlui.administrative.Navigation.context_export_community">Export Community</message>
         <message key="xmlui.administrative.Navigation.context_export_metadata">Export Metadata</message>
+    <message key="xmlui.administrative.Navigation.context_liveimport">Elsevier Import</message>
```

Comment:
I completely agree, perhaps I forgot to mention it much sooner. It would be misleading to call it Elsevier import and include only ScienceDirect, not Scopus.